test/run-souffle-tests.py

```diff
@@ -98,7 +98,9 @@
     "sort",             # 202
     "tak",              # 197
     "tic-tac-toe",      # 197
-    "weighted_distances" # 197
+    "weighted_distances", # 197
+    "multiple_inequalities",
```

Comment:
you will have to delete these if you merge the other pr

Reply:
Ok, will make sure to remember this when rebasing the other PR.